dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -1444,54 +1828,44 @@ RtpsUdpDataLink::send_heartbeat_replies() // from DR to DW
       {0 /* acknack count */}
     };
 
-    size_t size = 0, padding = 0;
-    gen_find_size(acknack, size, padding);
-    acknack.smHeader.submessageLength =
-      static_cast<CORBA::UShort>(size + padding) - SMHDR_SZ;
-    InfoDestinationSubmessage info_dst = {
-      {INFO_DST, FLAG_E, INFO_DST_SZ},
-      {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
-    };
-    gen_find_size(info_dst, size, padding);
-
-    ACE_Message_Block mb_acknack(size + padding); //FUTURE: allocators?
-    // byte swapping is handled in the operator<<() implementation
-    Serializer ser(&mb_acknack, false, Serializer::ALIGN_CDR);
-    std::memcpy(info_dst.guidPrefix, pos->writerid.guidPrefix,
-                sizeof(GuidPrefix_t));
-    ser << info_dst;
-    // Interoperability note: we used to insert INFO_REPLY submessage here, but
```

Comment:
We shouldn't lose this.  It could move to `docs/design/RTPS`.

Reply:
Which part? The code itself or the comment about INFO_REPLY?

Reply:
Just the comment about interoperability